ndcube/mixins/plotting.py

```diff
@@ -186,6 +185,12 @@ def _plot_2D_cube(self, wcs, axes=None, plot_axes=None, axes_coordinates=None,
     def _animate_cube(self, wcs, plot_axes=None, axes_coordinates=None,
                       axes_units=None, data_unit=None, **kwargs):
 
+        try:
+            from sunpy.visualization.animator import ArrayAnimatorWCS
```

Comment:
```suggestion
            from sunpy.visualization.animator import ArrayAnimatorWCS  # isort:skip
```
This should make the codestyle test ignore this import and hopefully pass.

Reply:
I thought I saw all the tests pass without this, but it doesn't hurt.